tensorflow/python/ops/sparse_ops.py

```diff
@@ -1066,7 +1066,7 @@ def sparse_split_v2(sp_input=None,
   >>> indices = [[0, 2], [0, 4], [0, 5], [1, 0], [1, 1]]
   >>> values = [1, 2, 3, 4, 5]
   >>> t = tf.sparse.SparseTensor(indices=indices, values=values,
-  dense_shape=[2, 7])
+  ... dense_shape=[2, 7])
```

Comment:
Nit: Can you fix indentation here?

Reply:
Done. This doesn’t seem to be applied consistently though, see e.g. [here](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/ops/sparse_ops.py#L1288).